spark/src/main/scala/io/substrait/spark/logical/ToLogicalPlan.scala

```diff
@@ -193,6 +196,27 @@ class ToLogicalPlan(spark: SparkSession) extends DefaultRelVisitor[LogicalPlan]
     }
   }
 
+  override def visit(expand: relation.Expand): LogicalPlan = {
+    val child = expand.getInput.accept(this)
+    val names = expand.getHint.get().getOutputNames.asScala
```

Comment:
What happens if this is not set? Can you generate names for this on the fly?

I'm okay with leaving this as is for now, but generally when consuming a plan hints should be entirely optional for producers to set.